build.sbt

```diff
@@ -132,17 +132,6 @@ lazy val amm = project
     // classpaths, so that the `source` macro can find their sources and
     // highlight/display them.
```

Comment:
This comment used to apply to the code you moved, which is now a bit far away. Perhaps you could split the comment in two and have the relevant text near the relevant piece of code?

Reply:
Yep, fixed that in the third commit.